pages/dkp/kommander/2.1/backup-and-restore/index.md

```diff
@@ -197,6 +197,46 @@ To restore cluster data on demand from a selected backup snapshot available in t
 velero restore create --from-backup BACKUP-NAME
 ```
 
+#### Restoring Kommander Management Cluster
+
+<p class="message--note"><strong>NOTE: </strong>When restoring a backup to a Kommander Management Cluster that is running Kommander installation it is necessary to adjust configuration to avoid restore errors.
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>When restoring a backup to the Kommander Management Cluster, you must adjust configuration to avoid restore errors.
```

Or do we call the management cluster “Host Cluster” in docs? (I think this is inconsistent across docs, maybe we should pick one and change it everywhere)

Reply:
The modified page is using the phrase "Kommander Management Cluster".